be/src/olap/data_dir.cpp

```diff
@@ -267,12 +281,13 @@ Status DataDir::_init_meta() {
     _meta = new (std::nothrow) OlapMeta(_path);
     if (_meta == nullptr) {
         LOG(WARNING) << "new olap meta failed";
-        return Status::InternalError("new olap meta failed");
+        return Status::MemoryAllocFailed("new olap meta failed");
```

Comment:
Why not change it to `RETURN_NOT_OK_STATUS_WITH_WARN`?

Reply:
Updated.